email/message/src/main/java/com/blazebit/notify/email/message/EmailNotificationMessageResolverFactory.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     public NotificationMessageResolver<EmailNotificationMessage> createNotificationMessageResolver(NotificationJobContext jobContext, ConfigurationSource configurationSource) {
-        return new EmailNotificationMessageResolver(jobContext, configurationSource);
+        return new EmailNotificationMessageResolver(jobContext, configurationSource, null);
```

Comment:
In the constructor, a `List<NotificationMessageResolverModelCustomizer>` should be initialized via `ServiceLoader.load(NotificationMessageResolverModelCustomizer.class)` and passed here